tests/python/test_python_frontend.py

```diff
@@ -3123,6 +3123,26 @@ def fusion_func(fd: FusionDefinition) -> None:
         torch_ref = inputs[0] * (inputs[1] * inputs[2]).unsqueeze(-1)
         self.assertEqual(nvf_out[0], torch_ref)
 
+    def test_issue2755(self):
```

Comment:
Can we add a small TODO that we expect this test to fail with `wjy/input` but pass on ToT main?